LoopUI/Extensions/ChartAxisValuesStaticGenerator.swift

```diff
@@ -33,8 +33,9 @@ extension ChartAxisValuesStaticGenerator {
             
             /// The first axis value will be less than or equal to the first scalar value, aligned with the desired multiple
             var firstValue = first - (first.truncatingRemainder(dividingBy: multiple))
-            /// The last axis value will be greater than or equal to the first scalar value, aligned with the desired multiple
-            var lastValue = last + (abs(multiple - last).truncatingRemainder(dividingBy: multiple))
+            /// The last axis value will be greater than or equal to the last scalar value, aligned with the desired multiple
+            let remainder = last.truncatingRemainder(dividingBy: multiple)
```

Comment:
I'll have to trust you on the math here.

Reply:
:) my math is trustworthy... but I'll put in tests as well.